src/rules/n/no-extraneous-import.d.ts

```diff
@@ -0,0 +1,73 @@
+import type { RuleConfig } from '../rule-config';
+
+/**
+ * Option.
+ */
+export interface NoExtraneousImportOption {
+  allowModules?: string[];
+  convertPath?:
+    | {
+        /**
+         * @minItems 2
+         * @maxItems 2
+         *
+         * This interface was referenced by `undefined`'s JSON-Schema definition
+         * via the `patternProperty` "^.+$".
```

Comment:
I personally go manually through the generated files and remove some garbage manually afterwards
Still not satisfied with my process, but this is coming from a JSONSchema generator sub dependency :confused:

Reply:
Dude, if that's the only thing we have to do, that's awesome. 99% of the boring work is done automatically. Then we can remove that thing automatically pretty easily too

Reply:
> Dude, if that's the only thing we have to do, that's awesome. 99% of the boring work is done automatically. Then we can remove that thing automatically pretty easily too ( I guess ? )

Exactly :heart: